packages/message-relayer/src/service.ts

```diff
@@ -197,6 +197,17 @@ export class MessageRelayerService extends BaseService<MessageRelayerOptions> {
             { batchSize: size }
           )
           this.state.nextUnfinalizedTxHeight += size
+
+          // Only deal with ~1000 transactions at a time so we can limit the amount of stuff we
+          // need to keep in memory. We operate on full batches at a time so the actual amount
+          // depends on the size of the batches we're processing.
+          if (
```

Comment:
This style is incredibly verbose

```js
const shouldBreak = this.state.nextUnfinalizedTxHeight - this.state.lastFinalizedTxHeight > 1000
if (shouldBreak) {
   ...
}
```
Could break up the line with the subtraction into 2 lines since it'll probably break the linter

Reply:
100x easier to read now, thank you